java/src/jmri/jmrix/can/cbus/swing/cbusslotmonitor/CbusSlotMonitorDataModel.java

```diff
@@ -87,7 +91,9 @@ protected TextAreaFIFO tablefeedback(){
         ("Any Functions set to ON"),
         ("Speed Steps"),
         null, // consist id
-        null // flags
+        null, // flags
+        Bundle.getMessage("ReleaseTip"),  // send KLOC
```

Comment:
What does KLOC stands for?

Reply:
It's a CBUS OPC for Kill Loco session, I've used Release rather than Kill in the GUI as it's a bit friendlier !